pkg/operator/vitesstopo/register.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2019 PlanetScale Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vitesstopo
+
+import (
+	"context"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/topo"
+
+	v2 "planetscale.dev/vitess-operator/pkg/apis/planetscale/v2"
+	"planetscale.dev/vitess-operator/pkg/operator/lockserver"
+	"planetscale.dev/vitess-operator/pkg/operator/results"
+)
+
+const (
+	// topoRequeueDelay is how long to wait before retrying when a topology
+	// server call failed. We typically return success with a requeue delay
+	// instead of returning an error, because it's unlikely that retrying
+	// immediately will be worthwhile.
+	topoRequeueDelay = 5 * time.Second
+)
+
+type RegisterCellsCmd struct {
```

Comment:
Would you be ok if we called this RegisterCellsParams to match precedent in our operator of VitessLockserverParams?

Reply:
Sure - I don't feel strongly about that so I can update the name. I choose `Cmd` (short for command) because command objects are parameter objects that are used to execute commands (mutate state) so it was the first thing that came to mind.

Reply:
Updated https://github.com/planetscale/vitess-operator/pull/48/commits/aadda76b17e732fa36b501e312d595246689d91c.